index.js

```diff
@@ -4,10 +4,14 @@ const electron = require('electron');
 const app = electron.app;
 const Menu = electron.Menu;
 const BrowserWindow = electron.BrowserWindow;
+const { globalShortcut } = require('electron')
```

Comment:
I don't know TS very well. Can you explain what the difference is between line 6 and 7? Obviously they are syntactically different; however, do they accomplish the same thing?

I'm curious if lines 3-7 may be changed to:
const { app, Menu, BrowserWindow, globalShortcut } = require('electron')?

Reply:
They are equivalent ways of assignment, "const { globalShortcut } = require('electron')" is just shorter . I have updated the code so it uses const { app, Menu, BrowserWindow, globalShortcut } = require('electron'). Thanks

Reply:
Ok, sounds good. Thank you for the clarification.